courseraprogramming/commands/upload.py

```diff
@@ -347,6 +356,27 @@ def parser(subparsers):
              'also be associated with the grader.')
 
     parser_upload.add_argument(
+        '--grader-cpu',
+        type=int,
+        choices=[1, 2],
+        help='Amount of CPU your grader is allocated. You may choose from 1 '
```

Comment:
We should probably specify that this is the amount of CPU the grader is allocated when grading learner submissions to avoid confusion regarding whether these are the resources that are used to process the grader upload. Similarly belowe for memory.